tests/spread/rockcraft/base-devel/task.yaml

```diff
@@ -31,7 +31,7 @@ execute: |
   rm base-devel_0.1_amd64.rock
   docker images base-devel:0.1
   id=$(docker run --rm -d base-devel:0.1)
-  test "$(docker inspect "$id" -f '{{json .Config.Entrypoint}}')" = '["/.rock/bin/pebble","enter"]'
+  test "$(docker inspect "$id" -f '{{json .Config.Entrypoint}}')" = '["/usr/bin/pebble","enter"]'
```

Comment:
since we have this test can you add a check that `docker exec "$id" pebble services" works?

Reply:
Done